mkdocs.yml

```
@@ -0,0 +1,47 @@
+
+markdown_extensions:
+    - markdown_include.include:
+        base_path: docs
+    - admonition
+    - toc:
+        permalink: ""
+
+extra_css:
+    - css/custom.css
+extra_javascript:
+    - js/custom.js
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 4

(indentation)